### DISCUSSION TITLE
Contributor incentive plan detail discuss

### DISCUSSION THREAD

**Discussion by Pana** [Conflux-Chain/conflux-documentation#243] [General]
Discuss and determine a detailed, feasible incentive plan to encourage more community developers to participate in document contributions. The plan needs to be clear and specific, for example, what actions will result in what kind of contributions.

**Comment by jackleeio** [comment]


**Reply by jackleeio** [reply]
Another suggestion is that I think it would be better to change the review time to once a month, which will help to activate the community @darwintree @Pana

**Reply by darwintree** [reply]
We will take the suggestion to shorten the review time into consideration as current situation varies from initial

**Comment by darwintree** [comment]
@jackleeio We are currently reviewing the rewards guideline and found out that a misunderstanding might occur relating to the pull requests. 

```
- Providing detailed elaboration on a topic: From 60 FC, up to 300 FC or more, determined by content and significance.
- Crafting new articles on a Conflux topic, depending on complexity and community need: 300 FC, potentially more.
```

As for the latter one, we were intending to refer to "Conflux-specific topic" which does not match your recent pull requests. But as for the inaccurate statement, we are not sure if you've understood the guideline. If it were the case, we would reward you as 300 FC for the recent new articles added. And if there were not a misunderstanding, we would issue rewards as original intended.

**Reply by darwintree** [reply]
commented below⬇️

**Comment by darwintree** [comment]
> The former is the best situation I thought before, and the latter is the worst situation I think. I also think the official incentives will be more generous in the early days when there are not many people contributing.
> 
> If you mean the total reward is only 300FC, I will not waste any more time on this, you're treating the time of community developers as if it's of little value. I spent a total of about 100 hours on this, and got 300FC, the hourly wage is even lower than that of McDonald's, it's practically a joke.🤣

Sorry for that I did not express the problem clearly. I'm making sure if there does exist a misleading, we'll still distribute rewards as 300 FC for **each** new article created( sorry the each is missed from the comments). I would give you the current estimation of the 2 situations I mentioned. 
1. The intended rewards we would distribute: improvements are evaluated case by case (as mentioned at least 60 FC). And as for you articles, each article will range from 150-300, based on the content complexity, and currently sum up to over 3000 FC(an estimation currently).
2. The way you are misunderstanding: 300 FC for each article. The final reward will sum up to over 4000 FC. (Anyhow, as there exist a misunderstanding, you will get 300FC for each article)

We're setting the rule because we would like contributors to create more contents specific for Conflux. And it is also As for the general part, developers could also refer to general documents on the internet for help.

If you are considering the value suggested by guideline is not resonable, please let us know what's your expectation and we would love to hear your voice

**Reply by jackleeio** [reply]
There were indeed some misunderstandings before. I had always thought the reward was 300FC per article, but I was quite surprised yesterday when I thought the total reward was only 300FC.

> We're setting the rule because we would like contributors to create more contents specific for Conflux. And it is also As for the general part, developers could also refer to general documents on the internet for help.

If you think that general documents are not as important and decide to apply the first set of rules below, and use the second set for Conflux-specific documents, I can actually accept that as well. There were some misunderstandings before, as I initially thought that general documents were also part of Conflux topics.

```
- Providing detailed elaboration on a topic: From 60 FC, up to 300 FC or more, determined by content and significance.
- Crafting new articles on a Conflux topic, depending on complexity and community need: 300 FC, potentially more.
```

However, regarding what you mentioned, I have a suggestion about the general part you talked about.

A good document should not only assume that developers are experienced but should consider if they are beginners or people who are touching blockchain development for the first time. We should think about how Conflux documentation can help them, rather than simply saying developers can find more information elsewhere. 

We should believe that more developers from Web2 will enter Web3 through Conflux in the future. If that's the case, then Conflux's documentation could serve as a great learning and advancement guide, attracting new developers entering Web3 to try building applications on Conflux. On this point, I think we should think more long-term, good general documentation will make Conflux more competitive in attracting non-Web3 developers.

So in the long run, general part is also very important, having the same weight for rewards in this part as for the Conflux-specific topics would be better, at least the difference shouldn’t be too far apart.

**Reply by darwintree** [reply]
Yes, I agree with the idea that we also need documents for beginners and this is quite what we are focused on in the previous months. However, as this is the website for Conflux, and giving that we only have limited resource, we would always put contents relating to Conflux to a higher place, and that is why a higher floor value was set for conflux-specific articles. I think #435 is a perfect example illustrating what we need: it provides tutorials for beginners as well as letting them to build for Conflux.

As stated in #507 , we would apply some changes to our reward guideline in the new quarter. We would provide more detailed guideline for the general contents created so you will know you won't receive only 80 FC if you want to create awesome contents. The new pull request for the change will come out soon and welcome to comment

**Reply by jackleeio** [reply]
Okay, I will try to create some conflux-specific content by referring to the #435 example.

**Comment by jackleeio** [comment]
<img width="521" alt="Screenshot 2024-06-11 at 12 52 12" src="https://github.com/Conflux-Chain/conflux-documentation/assets/20477587/f9354d07-de88-4d13-b279-0cd0736ade79">


At this stage, the price of FC has dropped by two-thirds (from 0.9u to 0.3u) since the last review. Despite this, the time required to create tutorials has not decreased. Additionally, FC’s liquidity is worsening, with diminishing liquidity in DEX pools, and Swappi does not support FC. I propose increasing the incentives for creation appropriately.

@Pana  @darwintree

**Reply by darwintree** [reply]
The reward value was initially decided to stay stable regardless of market price changes. But thank you for your feedback and we would still value your opinion and seek solutions for drastic price changes

**Reply by darwintree** [reply]
We would know if you are willing to publish the reviewed articles in our forum, under which case independent rewards can be distributed. You can email build@confluxnetwork.org if you would know more info
